README.md

```diff
@@ -1 +1,2 @@
-# Review-one-of-your-past-projects1
\ No newline at end of file
+# Review-one-of-your-past-projects1
+Copied for review
```

Comment:
Hi Zuz4n!
Good job so far! but there are some issues that you still need to work on, on Readme file please be sure to had a professional view, here is an example how to do that.

![](https://img.shields.io/badge/Microverse-blueviolet)

# Project Name

> Description the project.


## Built With

- Major languages
- Frameworks
- Technologies used

## Live Demo (if available)

[Live Demo Link](https://livedemo.com)


## Getting Started

**This is an example of how you may give instructions on setting up your project locally.**
**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**


To get a local copy up and running follow these simple example steps.

### Prerequisites

### Setup

### Install

### Usage

### Run tests

### Deployment



## Authors

👤 **Author1**

- GitHub: [@githubhandle](https://github.com/githubhandle)
- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)

👤 **Author2**

- GitHub: [@githubhandle](https://github.com/githubhandle)
- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)

## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Hat tip to anyone whose code was used
- Inspiration
- etc

## 📝 License

This project is [MIT](./LICENSE) licensed.

_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

Reply:
Good job for adding such a polite and descriptive comment 👌but don't you think it'll be better to provide a link to the README file template instead of putting the README file in the comment? This will make your review comment much more professional and short. Please make sure to edit your comment and replace the entire README file with the link.